go-controller/pkg/ovn/gateway.go

```diff
@@ -0,0 +1,67 @@
+package ovn
+
+import (
+	"os/exec"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+)
+
+func (ovn *Controller) getOvnGateways() ([]string, error) {
+	// Return all created gateways.
```

Comment:
I guess we should maintain a cache, otherwise it will be very slow to respond to services/endpoints.
Opening an issue to track this.